### PR TITLE
[SCT-593] NullPointerException running search_objects query

### PR DESCRIPTION
**Pull request by tiramisu24**
- account for elasticsearch returning null for highlight instead of empty map
- appears to occur somewhat consistently for GUIDs and sporadically for other search queries.

### CODE REVIEW

**Comment by tiramisu24** [comment]
Whats the best way to test this? Can I just write a test with one of the queries that highlight happens to return null?

**Comment by MrCreosote** [comment]
That's how I'd do it.

File: test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java [hunk 1]
```diff
@@ -445,6 +445,21 @@ public void highlightTest () throws Exception{
 
         //highlight in objects
         assertThat("incorrect highlight", actual.getHighlight(), is(expected.getHighlight()));
+
```
Comment (MrCreosote) [inline test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java]: This should be a unit test, not an integration test. In the interests of getting this out, do you solemnly swear to switch it to an unit test when the API changes are done?

Comment (tiramisu24) [reply test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java]: yes :)

**Pull request merged**